lib/enumerize/base.rb

```diff
@@ -6,6 +6,14 @@ def self.included(base)
       if base.respond_to?(:validate)
         base.validate :_validate_enumerized_attributes
       end
+
+      class << base
+        if method_defined?(:inherited) && (not method_defined? :inherited_without_enumerized)
+          alias_method :inherited_without_enumerized, :inherited
+        end
+
+        alias_method :inherited, :inherited_with_enumerized
```

Comment:
The first if condition (L11) is necessary for avoiding circular calling `inherited` when `include` is evaluated twice.
But the second `alias_method` (L15) is necessary to run each `include` evaluation because
the `add_dependant subclass.enumerized_attributes` is requied to called each inherited class/modules scope.

Such situation is happened when `Enumerize` moudle extented from other module, 
and concrete class includes it. 
It can see in following test code.

https://github.com/brainspec/enumerize/blob/master/test/activerecord_test.rb#L46

Actually, The seccond `alias_method` (L15) moves into the above if condition, 
Some test fails.

```
Finished tests in 0.436918s, 377.6454 tests/s, 601.9439 assertions/s.

  1) Failure:
Enumerize::Base#test_0014_inherits enumerized attributes from a parent class [/home/ozaki/dev/enumerize/test/base_test.rb:111]:
Expected nil to be an instance of Enumerize::Attribute, not NilClass.

  2) Failure:
Enumerize::Base#test_0017_adds new parent class attributes to subclass [/home/ozaki/dev/enumerize/test/base_test.rb:129]:
Expected nil to be an instance of Enumerize::Attribute, not NilClass.

  3) Failure:
Enumerize::Base#test_0015_inherits enumerized attributes from a grandparent class [/home/ozaki/dev/enumerize/test/base_test.rb:116]:
Expected nil to be an instance of Enumerize::Attribute, not NilClass.

165 tests, 263 assertions, 3 failures, 0 errors, 0 skips
```